index.php

```diff
@@ -11,6 +11,20 @@
 	</script>
 	<meta name="google" value="notranslate" />
 	<meta http-equiv="Content-type" content="text/html; charset=utf-8" />
+	<script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
```

Comment:
Wow,
Thanks for all the input. I didn't even mean to submit the pull request :)

On Fri, Jul 15, 2016 at 12:14 PM, jangellx <notifications@github.com> wrote:

> In index.php
> <https://github.com/jangellx/MagicMirror/pull/2#discussion_r70999172>:
>
> > @@ -11,6 +11,20 @@
> >  	</script>
> >  	<meta name="google" value="notranslate" />
> >  	<meta http-equiv="Content-type" content="text/html; charset=utf-8" />
> > +	<script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
>
> I'd probably move this to main.js. The username to monitor should also be
> defined in config.js, as should the limit.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/jangellx/MagicMirror/pull/2/files/ca5d9e01daa43d205e09a26093ce79358187092e#r70999172>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHYb2hIWtHEhqEbldFEGpIW21uNFhmFpks5qV7HWgaJpZM4JNinf>
> .
>